notebooks/whisper-subtitles-generation/whisper-subtitles-generation.ipynb

```diff
@@ -13,7 +13,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.            if segment.end_ts is None:
</pre><p>I think it should be:</p><p><code>if segment.end_ts == -1</code></p><p>according to <a href="https://github.com/openvinotoolkit/openvino.genai/pull/910/files#diff-57b99dea7bc9b9a0b5b94b77a150999f0be5aefd4f9ce0bfd8c758fbaf8b2de9R27" target="_blank">https://github.com/openvinotoolkit/openvino.genai/pull/910/files#diff-57b99dea7bc9b9a0b5b94b77a150999f0be5aefd4f9ce0bfd8c758fbaf8b2de9R27</a></p><p>You can reproduce it by cutting audio in the middle of the first expected chunk with setting low max_new_tokens. <a href="https://github.com/openvinotoolkit/openvino.genai/pull/910/files#diff-ae43f5490434ac5806d8f435291d94268c759ec8b26cc93cbeacdbb5d31f8c13R419" target="_blank">https://github.com/openvinotoolkit/openvino.genai/pull/910/files#diff-ae43f5490434ac5806d8f435291d94268c759ec8b26cc93cbeacdbb5d31f8c13R419</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2431/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>changed</p>